src/main.rs

```diff
@@ -197,8 +197,8 @@ fn get_setup(args: &[String]) -> Setup {
     const AP_PORT: &str = "ap-port";
     const AUTOPLAY: &str = "autoplay";
     const BACKEND: &str = "backend";
-    const BITRATE: &str = "b";
-    const CACHE: &str = "c";
+    const BITRATE: &str = "bitrate";
+    const CACHE: &str = "cache";
```

Comment:
These were the only ones that were the short version of the options/flags (b, c) I changed them for the sake of consistency. I also changed help for the same reason. I fixed them in 

Old (inconsistent):
https://github.com/librespot-org/librespot/blob/dev/src/main.rs#L200-L201
https://github.com/librespot-org/librespot/blob/dev/src/main.rs#L213
https://github.com/librespot-org/librespot/blob/dev/src/main.rs#L240-L248
https://github.com/librespot-org/librespot/blob/dev/src/main.rs#L263-L267

New (consistent):
https://github.com/librespot-org/librespot/blob/f353759b31654388a884e0b5b063f0b387428afd/src/main.rs#L240-L243
https://github.com/librespot-org/librespot/blob/f353759b31654388a884e0b5b063f0b387428afd/src/main.rs#L301-L305
https://github.com/librespot-org/librespot/blob/f353759b31654388a884e0b5b063f0b387428afd/src/main.rs#L319-L323

You'll also notice that I changed the order to all flags and then options and each section starts with the ones with short versions so that the help menu looks nicer IMHO.